tests/integration/components/moderators/component-test.ts

```diff
@@ -142,4 +143,79 @@ module('Integration | Component | moderators', hooks => {
         assert.dom('[data-test-permission-group]').exists({ count: 1 });
         assert.dom('[data-test-delete-moderator-button]').doesNotExist();
     });
+
+    test('can add moderator by searching for user', async function(
```

Comment:
It would be nice to add some tests for validation and some Percy snapshots.

Reply:
Since this is a integration test, I think the best is to reserve Percy snapshots for acceptance tests.